edit/management.cattle.io.project.vue

```diff
@@ -58,6 +59,24 @@ export default {
       return canViewProjectMembershipEditor(this.$store);
     },
 
+    canEditProject() {
+      const projectOptions = this.$store.getters['type-map/optionsFor'](MANAGEMENT.PROJECT);
+
+      return !!(this.value?.links?.update && projectOptions && projectOptions.isEditable);
```

Comment:
@richard-cox since we "hacked" the `canUpdate` on the model side, we need this to make a distinction between the users who can really edit a project or not